docs/tutorials/fundamentals/installation.md

```diff
@@ -170,6 +170,13 @@ the current release {{ napari_version }}, using command: `napari --version` .
 ````
 ![macOS desktop with a napari viewer window without any image opened in the foreground, and a terminal in the background with the appropriate conda environment activated (if applicable) and the command to open napari entered.](../assets/tutorials/launch_cli_empty.png)
 
+````{note}
+On some platforms, particularly macOS and Windows, there may be a ~30 second 
+delay before the viewer appears on first launch. This is normal and subsequent
```

Comment:
I don't know if I woudl call it "normal", but I get your point xD

Reply:
Would you prefer "common" ?

Reply:
I would rather point out that this is an *issue* with those platforms, cause as it is it sounds like there's not expectation/hope of this to get fixed.